mailbox/store/src/main/java/org/apache/james/mailbox/store/extractor/JsoupTextExtractor.java

```diff
@@ -38,6 +38,9 @@
 
     @Override
     public ParsedContent extractContent(InputStream inputStream, String contentType) throws Exception {
+        if (inputStream == null) {
```

Comment:
you mix early return and fallback style return.
Could you please use a single style?